_docs/compliance/domain-standards.md

```diff
@@ -40,7 +40,7 @@ TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384 (0xC028)
 
 These are false positives. While the CBC cipher modes of operation are being phased out (they are theoretically subject to padding oracle attacks), our cipher implementation is consistent with all relevant guidance, as noted above.
 
-**TLS 1.3**: Where we have not implemented TLS 1.3, it's because the underlying AWS service does not yet support it.
+**TLS 1.3**: Where we have not implemented TLS 1.3, it's because the underlying AWS service does not yet support it. In March, 2023, [AWS announced TLS 1.3](https://aws.amazon.com/about-aws/whats-new/2023/03/application-load-balancer-tls-1-3/) availability for AWS GovCloud, so cloud.gov will pursue implementation by mid-2023.
```

Comment:
This is fine by me; let's just make sure there's an GH issue for it or it's brought up at PI planning (where we'll then make an issue for it).